recipes/ruby/meta.yaml

```diff
@@ -1,11 +1,11 @@
 # Based on channel tswicegood
```

Comment:
Maybe it would help to switch to the recipe from the ruby-lang channel:
https://anaconda.org/ruby-lang/ruby

They seem to have an osx build.